test/games/strategy/engine/message/RemoteMessengerTest.java

```diff
@@ -30,7 +31,7 @@
   @Before
   public void setUp() throws Exception {
     // simple set up for non networked testing
-    m_messenger = new DummyMessenger();
+    m_messenger = MockObjects.getDummyMessenger();
```

Comment:
There is a lot of value in having self-contained tests. Avoiding external test  data or setup is very useful when debugging test failures.